components/hero.js

```diff
@@ -7,13 +7,13 @@ const Title = glamorous.h1((props, {colors, mediaQueries}) => ({
   color: colors.primaryMed,
```

Comment:
I reduce the size of the title, to fit the code editor without scrolling on a 13'. the main reason is to let the visitor see the whole code instantly.
I think the live editor is the "sexy" part of the homepage, so it should be see quickly and accessible too.

I reduce the logo but i didn't want to reduce him more so i reduce the title accordingly to keep the proportions and let the logo be the biggest thing in the <Hero />.